src/lib/utils/hooks/SynapseAPI/oauth/useOAuthClient.ts

```diff
@@ -43,11 +37,7 @@ export function useDeleteOAuthClient(
     {
       ...options,
       onSuccess: async (updatedClient, clientId, ctx) => {
-        await queryClient.invalidateQueries(oAuthQueryKeys.all(accessToken!))
-        queryClient.setQueryData(
-          oAuthQueryKeys.edit(accessToken!, clientId),
-          updatedClient,
-        )
+        await queryClient.invalidateQueries()
```

Comment:
See above comment. Just invalidate the `all`/`list` key. I think invalidating everything is more heavy handed than you need to be.